### PR TITLE
hancom-office 12.30.0.5542 (new cask)

### PR DESCRIPTION
**Pull request by leejongyoung**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/h/hancom-office.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask "hancom-office" do
+  version "12.30.0.5542"
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/h/hancom-office.rb:3]: The `url` is versioned, so there should be a `sha256`.

File: Casks/h/hancom-office.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  url "https://cdn.hancom.com/pds/hnc/DOWN/HancomOffice-general-#{version}.pkg",
+      referer: "https://hancom.com/mypage/myIndivPrdtList.do"
+  name "Hacom Office for Mac"
+  desc "Word processor"
+  homepage "https://office.hancom.com/"
+
+  livecheck do
+    skip "No version information available"
+  end
```
Comment (bevanjkay) [inline Casks/h/hancom-office.rb:13]: Please try to find a working `livecheck`

File: Casks/h/hancom-office.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  pkg "HancomOffice-general-#{version}.pkg"
+
+  uninstall quit:    "com.hancom.office.hwp#{version.major}.mac.general",
+            pkgutil: "com.hancom.office.hwp#{version.major}.mac.general"
+
+  zap trash: [
+    "/private/var/db/receipts/com.hancom.office.hwp#{version.major}.mac.general.bom",
+    "/private/var/db/receipts/com.hancom.office.hwp#{version.major}.mac.general.plist",
```
Comment (bevanjkay) [inline Casks/h/hancom-office.rb:22]: These items can be removed as they are temporary files.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by bevanjkay** [comment]
Unfortunately we can't use the extract plist strategy here because it checks the contents of the downloaded files. So in this case it will always return the current cask version because the download is versioned.

**Comment by samford** [comment]
Do you know of any public pages that list version information for this software? I don't see any version information, download links, etc. on https://office.hancom.com/. The only version information I saw when looking around was https://www.hancom.com/cs_center/csDownload.do and those versions don't correspond to the version used in the cask.

How did you find this pkg file?

**Pull request closed without merging**